src/infra/core/Cargo.toml

```diff
@@ -32,15 +32,16 @@ query-extensions-json = ["dep:datafusion-functions-json"]
 
 [dependencies]
 # Kamu
-internal-error = { workspace = true }
 container-runtime = { workspace = true }
-kamu-data-utils = { workspace = true }
-opendatafabric = { workspace = true }
+internal-error = { workspace = true }
 kamu-accounts = { workspace = true }
-kamu-datasets = { workspace = true }
+kamu-auth-rebac = { workspace = true }
```

Comment:
Try not to insert this dependency, and similar in dev version.

Reply:
Fixed